src/Analyzer.Core.BuiltInRuleTests/Tests/TA-000032/TA-000032.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "Template": "APIMService.json",
+    "ReportedFailures": [
+      {
+        "LineNumber": 50,
+        "Description": "Validate Certificate Chain is False"
```

Comment:
Is it usually `False` or `false` or does not matter?

Reply:
Updated to false